integration/tests/iam_v2.sh

```diff
@@ -32,14 +32,25 @@ hab_curl() {
 
 do_test_deploy() {
     log_info "Adding v1 policy (that should be migrated)"
-    local token
+    local token output
     token=$(chef-automate admin-token)
     hab_curl --fail -s -k -H "api-token: $token" \
       -d "$(jo subjects="$(jo -a user:local:alice team:local:ops)" action=read resource=auth:users)" \
       https://localhost/api/v0/auth/policies
 
     log_info "run chef-automate iam upgrade-to-v2"
-    chef-automate iam upgrade-to-v2 || return 1
+    if ! output=$(chef-automate iam upgrade-to-v2); then
+        log_error "Non-zero exit code, output:"
+        log_error "$output"
+        return 1
+    else
+        # in CI, we don't want any policies to be skipped
```

Comment:
This will catch _"unknown well-known"_ policies creeping in again in the future. I've ensured that the test works by [running it on master in this build](https://buildkite.com/chef/chef-automate-master-verify-private/builds/2168#72f4fcc7-43f4-4022-a8ea-ab5d09c1ff80/578-581)